site/content/en/docs/Guides/Client SDKs/_index.md

```diff
@@ -221,13 +221,109 @@ in [SetLabel(...)](#setlabelkey-value) above. The isolation is also important as
 Setting `GameServer` annotations can be useful if you want information from your running game server process to be 
 observable through the Kubernetes API.
 
+### Counters And Lists
+
+{{< alpha title="Counters And Lists" gate="CountersAndLists" >}}
+
+The `Counters` and `Lists` features in the SDK offer a flexible configuration for tracking various entities like 
+players, rooms, and sessions.
+
+{{< alert title="Note" color="info">}}
+The SDK batches mutation operations every 1 second for performance reasons. However, changes made and subsequently 
+retrieved through the SDK will be atomically accurate through the SDK, as those values are tracked within the 
+SDK Server sidecar process.
+
+However, changes made through Allocation or the Kubernetes API to `GameServer` List and Counter values will be 
+eventually consistent when being retrieved through the SDK.
+
+Since the SDK update operations back to the `GameServer.status` values is batched asynchronous, it is worth noting that
+any value incremented past a counter or list capacity may be silently truncated to the currently set capacity if 
+modified concurrently through the SDK and Allocation/Kubernetes API.
+{{< /alert >}}
+
+#### Counters
+
+All functions will return an error if the specified `key` is not predefined in the `GameServer.spec.counters` resource
+configuration.
+
+##### Alpha().GetCounterCount(key)
+
+This function retrieves the current count for a counter in the game server.
+
+##### Alpha().SetCounterCount(key, amount)
+
+This function sets the count to a given value. This operation overwrites any previous values and the new value cannot
+exceed the Counter's capacity.
+
+##### Alpha().IncrementCounter(key, amount)
+
+This function increments the count of a counter by a given non-negative value amount. The function returns an
+error if the Counter is already at capacity (at time of operation), indicating no increment will occur.
+
+##### Alpha().DecrementCounter(key, amount)
+
+This function decreases the count of a counter by a given non-negative amount. It returns an error if the
+Counter's count is already at zero.
+
+##### Alpha().SetCounterCapacity(key, amount)
+
+This function sets the maximum capacity for a counter. A capacity value of 0 indicates no capacity limit.
+
+##### Alpha().GetCounterCapacity(key)
+
+This function retrieves the maximum capacity of a counter.
+
+#### Lists
+
+All functions will return an error if the specified `key` is not predefined in the `GameServer.spec.lists` resource
+configuration.
+
+##### Alpha().AppendListValue(key, value)
+
+This function appends the specified string value to a List's values.
+
+An error is returned if the string already exists in the list or if the list is at capacity.
+
+##### Alpha().DeleteListValue(key, value)
+
+This function removes the specific string value string from a List's values. 
+
+An error is returned if the string does not exist in the list.
```

Comment:
(not for this release, just food for thought) - depending on feedback, we may want to revisit some of the handling of thing like returning an error if the value is already in the list. I feel like a non-error return of `false` (or language equivalent) in this scenario is a better developer experience, since we can't do very good cross platform enumeration of different types of errors? 🤔

Reply:
I'm thinking it would probably make most sense just to move this all over to the specific Go SDK reference, since that's really what these methods are.

Reply:
We always have a high level reference that is (generally) the basis for all language SDK guides. This makes building language agnostic integration guides easier, and general concept onboarding also easier, since we don't have to duplicate as much language across language SDK references.

Reply:
Or are you saying, let's just move the existing test over to the Go SDK? (now I read your comment again @igooch )

My point was more - we might want to tweak the Go SDK itself in terms of how it does error handling vs boolean returns, which may impact how the backing gRPC/REST client operates. But like I said - not for this release, just something to think about. I might file an issue for discussion after playing with this more once it's released.

Reply:
I'm talking about the reference guide modified in this PR, not any test. These methods are specific to the Go client.

Reply:
> These methods are specific to the Go client.

They are also specific to what will be built for the other language SDKs as well. That's why I made them more generic, so they could be used as the conceptual basis for pretty much all language SDKs.

The Go SDK will have it's own generated docs for it's own language specific reference.